GamaGameHub/GamaGameHub/appsettings.json

```diff
@@ -1,12 +1,20 @@
 {
   "ConnectionStrings": {
-    "DefaultConnection": "Server=(localdb)\\mssqllocaldb;Database=aspnet-GamaGameHub-17c66339-cd75-4595-a367-ea84292bee64;Trusted_Connection=True;MultipleActiveResultSets=true"
+    "DefaultConnection": "Server=KRESA\\SQLEXPRESS;Database=GamaGameHub;Trusted_Connection=True;MultipleActiveResultSets=true"
```

Comment:
This maybe should be left to something default, because otherwise every contributor will have to set it manually every time they need to. I'd even say we should include this file in the .gitignore and every contributor should set their file individually on their machine.